### PR TITLE
type: add id prop

### PR DESCRIPTION
**Pull request by li-jia-nan**
type: add id prop

### CODE REVIEW

**Comment by li-jia-nan** [comment]
- [https://github.com/ant-design/ant-design/pull/37392](https://github.com/ant-design/ant-design/pull/37392#discussion_r962481769)

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "rc-tree",
-  "version": "5.6.6",
+  "version": "5.6.7",
```
Comment (zombieJ) [inline package.json:3]: version 不用 bump~

File: src/TreeNode.tsx [hunk 2]
```diff
@@ -47,6 +47,7 @@ export interface TreeNodeProps<TreeDataType extends BasicDataNode = DataNode> {
   icon?: IconType;
   switcherIcon?: IconType;
   children?: React.ReactNode;
+  id?: string;
```
Comment (zombieJ) [inline src/TreeNode.tsx:50]: 挪到这里： https://github.com/react-component/tree/blob/0a8fe059bc82301f5ae8a5bdcc0cf891c9be7ecf/src/TreeNode.tsx#L19

File: package.json (hunk 1 above)
Comment (li-jia-nan) [reply package.json]: 已回滚

File: src/TreeNode.tsx (hunk 2 above)
Comment (li-jia-nan) [reply src/TreeNode.tsx]: 挪上去了

**Pull request merged**